cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2884,6 +2884,8 @@ Slice::Gen::InterfaceVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
         C << sp;
         C << nl << "const Ice::Current& current = request.current();";
         C << nl << "auto r = std::equal_range(allOperations.begin(), allOperations.end(), current.operation);";
+        // range is a C++ 20 feature and we want to keep the generated code C++17 compatible and lint-free.
```

Comment:
Do we run clang-tidy in C++17 or C++20 mode?

Reply:
.clang-tidy includes:
```
ExtraArgs: ['-std=c++17']
```

However, this specific test requires c++20:
```
$(project)_cppflags += -std=c++20
```

How this affects clang-tidy is not clear to me.